cli/cmd/root.go

```diff
@@ -143,3 +151,19 @@ func runVersionCheck(vChecker *version.VersionChecker) {
 		updateLastVersionCheck()
 	}
 }
+
+func isInstallSubCommand(flags []string) bool {
```

Comment:
TODO: Use a different variable name here. `flags` here means flags like `-q` and subcommands like `install` in a slice e.g., `[]string{"-q", "install"}`

Reply:
Done